iceoryx_posh/include/iceoryx_posh/internal/popo/ports/server_port_roudi.hpp

```diff
@@ -36,14 +37,16 @@ class ServerPortRouDi : public BasePort
   public:
     using MemberType_t = ServerPortData;
 
-    explicit ServerPortRouDi(cxx::not_null<MemberType_t* const> serverPortDataPtr) noexcept;
+    explicit ServerPortRouDi(MemberType_t& serverPortData) noexcept;
 
     ServerPortRouDi(const ServerPortRouDi& other) = delete;
     ServerPortRouDi& operator=(const ServerPortRouDi&) = delete;
     ServerPortRouDi(ServerPortRouDi&& rhs) noexcept = default;
     ServerPortRouDi& operator=(ServerPortRouDi&& rhs) noexcept = default;
     ~ServerPortRouDi() = default;
 
+    ConsumerTooSlowPolicy getClientTooSlowPolicy() const noexcept;
```

Comment:
One is the generic enum name and the other one gives more context to the enum since it is also used at different places. For me it's like having and `int64_t` as data type and calling the variable `myInt64`. It is technically correct but doesn't give context

Reply:
In the long-run it might be better to return the options to the reader with one method instead of multiple individual getter methods. Read only access can be "ensured" by const ref or copy (more expensive).

But it is not important now.

Reply:
@MatthiasKillat it might be necessary with the integration into RouDi and the discovery loop.